src/autocomplete.js

```diff
@@ -308,14 +332,12 @@ class Autocomplete extends Component {
     if (event.key && this.keyDownHandlers[event.key]) {
       this.keyDownHandlers[event.key].call(this, event)
     } else if (!['Shift', 'Meta', 'Alt', 'Control'].includes(event.key)) {
-      this.openMenu(() => {
-        this.highlightIndex()
-      })
+      this.openMenu()
```

Comment:
I moved that logic into `toggleMenu` so anytime the menu is opened it will handled highlighting.